docs/integrations/labext.md

```diff
@@ -0,0 +1,68 @@
+# Jupyter Lab Extension
+
+Now you can commit your Jupyter Notebook to [Jovian](https://jovian.ml?utm_source=docs) with just **One Click**.
+Make sure you’ve completed the [Installation](../user-guide/01-install.md) before reading further.
+
+## Using Jovian Jupyter Lab Extension
+
+<img src="https://i.imgur.com/uezsdYX.png" class="screenshot" alt="jupyter extension toolbar button">
+
+Once you have successfully installed jovian, a new button `Commit` will appear on the tool bar. When using `Commit` button for first time you'll be asked to provide an API key.
+
+<img src="https://i.imgur.com/jTvA0De.png" class="screenshot" alt="jovian api key prompt">
+
+You can get the API key at [Jovian](https://jovian.ml?utm_source=docs). Once you log in, just click on `API key` button, and the key will be copied to the clipboard.
+
+<img src="https://i.imgur.com/taLLUVd.png" class="screenshot" alt="jovian api key copy button">
+
+#### Valid API key
+
+If the key is valid you will be notified with the following alert.
+
+<img src="https://i.imgur.com/VYIvAk2.png" class="screenshot" alt="valid api key prompt">
+
+#### Error with API key
+
+If the entered API key is invalid you will get following error.
+
+<img src="https://i.imgur.com/Ot75GEP.png" class="screenshot" alt="api key error prompt">
```

Comment:
won't it highlight the border of the input box with red color saying its a invalid key? Like it notebook's couterpart

Reply:
I can change the code in Lab ext now, so it will make it similar like what we have in nbext.

Reply:
finish changing